includes/event/event-capabilities.php

```diff
@@ -240,6 +252,45 @@ private function has_edit_attendees( WP_User $user, Event $event ): bool {
 		return false;
 	}
 
+	/**
+	 * Evaluate whether a user can edit event title for a specific event.
+	 *
+	 * @param WP_User $user  User for which we're evaluating the capability.
+	 * @param Event   $event Event for which we're evaluating the capability.
+	 * @return bool
+	 */
+	private function has_edit_form_field( WP_User $user, Event $event, $cap ): bool {
+		$event_start         = $event->start();
+		$happening_now       = $event->start()->is_in_the_past() && ! $event->end()->is_in_the_past();
+		$now                 = new DateTimeImmutable( 'now', new DateTimeZone( 'UTC' ) );
+		$event_end_plus_1_hr = $event->end()->modify( '+1 hour' );
+
+		if ( self::EDIT_DESCRIPTION === $cap ) {
+			return true;
+		}
+
+		if ( $event_start > $now ) {
+			return true;
+		}
+
+		if ( $happening_now && ! $this->stats_calculator->event_has_stats( $event->id() ) ) {
+			return true;
+		}
+
+		if ( $happening_now && $this->stats_calculator->event_has_stats( $event->id() ) ) {
+			return ( self::EDIT_TITLE === $cap || self::EDIT_END === $cap );
+		}
+
+		if ( $event->end()->is_in_the_past() && $now < $event_end_plus_1_hr ) {
+			return ( self::EDIT_TITLE === $cap || self::EDIT_END === $cap );
```

Comment:
I think if it's possible to edit the title, it should also be possible to edit the description, because the title is more restricted then description. Also applies to the condition above.